<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinaDetails</class>
 <widget class="QWidget" name="LinaDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подробнее о Lina</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>861</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LINA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>861</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="LinaGif">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>190</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string>Изображение персонажа</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Опасная и хрупкая Lina легко свалит любого одинокого врага. Она поражает противников огнём и молнией, а каждое произнесённое ей заклинание увеличивает скорость её атаки, не давая выжить никому.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>861</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Ab_1_img">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="text">
     <string>Способность 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Ab_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Gragon Slave</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>841</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Способности</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Ab_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Strike</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ab_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Laguna Blade</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ab_2_img">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="text">
     <string>Способность 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Ab_3_img">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="text">
     <string>Способность 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>610</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
